src/DefaultContentManager.php

```diff
@@ -108,7 +109,7 @@ public function __construct(Serializer $serializer, ResourcePluginManager $resou
    */
   public function importContent($module) {
     $created = array();
-    $folder = drupal_get_path('module', $module) . "/content";
+    $folder = '/var/www/html/tourist-mannheim/config/staging/content';//drupal_get_path('module', $module) . "/content";
```

Comment:
With this new behavior, we can't use modules as content storage because all exported files will be stored into the config folder. So instead of the module name, could the method accept the folder name as a parameter?